docker/Dockerfile_commcarehq_base

```diff
@@ -17,6 +17,8 @@ RUN npm -g install \
 
 RUN echo '{ "allow_root": true }' > /root/.bowerrc
 
+RUN git config --global url."https://".insteadOf git://
```

Comment:
I'm a bit confused by this. The Dockerfile doesn't do any git clones. Any git references are coming from the your own checkout of the CommCare HQ source. If you want to change these then you should change them in your local repo.